### PR TITLE
Add docs for Exporting user info to Pardot

### PR DESCRIPTION
**Pull request by AnuradhaSK**
## Purpose

- Add docs for Exporting user info to Pardot

## Depends on PR
#1761

### CODE REVIEW

File: en/docs/learn/exporting-user-info-to-pardot.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    [Exporting User Data From WSO2 Identity Server](../learn/exporting-user-data-from-wso2-is.md).
+
+##Try it Out
+
+1. Login to [Pardot](https://pi.pardot.com/).
+2. Navigate to **Admin > Import > Prospects**.
+    ![import-prospect-in-paradot](../assets/img/tutorials/import-prospect-in-paradot.png)
+3. Click **+ Upload File** and choose the created `users.csv` file.
+    ![upload-file-in-paradot](../assets/img/tutorials/upload-file-in-paradot.png)
+4. Click **Next**, after the file get uploaded successfully.
```
Comment (gomathyk) [inline en/docs/learn/exporting-user-info-to-pardot.md:20]: Once the file gets uploaded successfully, click **Next**.

File: en/docs/learn/exporting-user-info-to-pardot.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    ![proceed-after-file-uploading-paradot](../assets/img/tutorials/proceed-after-file-uploading-paradot.png)
+5. Map the column headers to Pardot fields. Pardot suggests some field mappings.
```
Comment (gomathyk) [inline en/docs/learn/exporting-user-info-to-pardot.md:22]: to the

File: en/docs/learn/exporting-user-info-to-pardot.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    ![map-fields-in-pardot](../assets/img/tutorials/map-fields-in-pardot.png)
+6. Complete the **Campaign and Add Tags** and **Action** section according to your needs and confirm user import.
```
Comment (gomathyk) [inline en/docs/learn/exporting-user-info-to-pardot.md:24]: sections
according to your requirement. 

7.  Confirm user import.

Comment (AnuradhaSK) [reply en/docs/learn/exporting-user-info-to-pardot.md]: Fixed in https://github.com/wso2/docs-is/pull/1765/commits/fc98995b5b535d5cd7b9a5f5b91ca538745a6095

File: en/docs/learn/exporting-user-info-to-pardot.md (hunk 2 above)
Comment (AnuradhaSK) [reply en/docs/learn/exporting-user-info-to-pardot.md]: Fixed in fc98995

File: en/docs/learn/exporting-user-info-to-pardot.md (hunk 1 above)
Comment (AnuradhaSK) [reply en/docs/learn/exporting-user-info-to-pardot.md]: Fixed in fc98995

**Pull request merged**